samples/grid/griddemo.cpp

```diff
@@ -920,6 +935,73 @@ void GridFrame::SetColLabelVertAlignment( wxCommandEvent& WXUNUSED(ev) )
 }
 
 
+void GridFrame::SetCornerLabelHorizAlignment( wxCommandEvent& WXUNUSED(ev) )
```

Comment:
Actually after testing this I've realized that the way these menu commands work doesn't make any sense: the "horizontal" and "vertical" alignment are radio items, but selecting them somehow cycles through the horizontal/vertical alignment styles independently??

It looks like the new functions just mimic the existing ones, but I'd rather not do this unless I'm really missing something here.

Reply:
Indeed, corner alignment test just mimics existing similar tests for row and column alignment. I also found them somewhat confusing at first, but then realized, that it is a smart way of testing all alignment combinations with only two radio buttons. Anyway, I probably considered existing tests as 'authoritative' in some sense...

Reply:
Well, OK, this is not the highest priority, but it's really weird. It'd be nice to streamline all this somehow later. But for now let's merge this as it is, thanks.